modules/tfm/tfm/boards/src/log_memory_protection.c

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <cmsis.h>
+#include <tfm_spm_log.h>
+#include <mpu_armv8m_drv.h>
+#include <nrf.h>
+#include <nrf_peripherals.h>
+#include <array.h>
+
+
+void log_memory_protection_of_mpu(void)
+{
+	MPU_Type * mpu = (MPU_Type *)MPU_BASE;
+	SPMLOG_DBGMSG("ARM MPU configuration\r\n");
+	SPMLOG_DBGMSGVAL("CTRL", mpu->CTRL);
+
+	for(int i = 0; i < MPU_ARMV8M_NUM_REGIONS; i++) {
+		mpu->RNR = i;
+		SPMLOG_DBGMSGVAL("RNR", i);
+		uint32_t rbar = mpu->RBAR;
+		uint32_t rlar = mpu->RLAR;
+		uint32_t en = (rlar & MPU_RLAR_EN_Msk) >> MPU_RLAR_EN_Pos;
+
+		SPMLOG_DBGMSGVAL("  EN", en);
+		if(en) {
+			SPMLOG_DBGMSGVAL("  BASE", rbar & MPU_RBAR_BASE_Msk);
+
+			// The last address that is protected is limit + 31
```

Comment:
Comments should be using `/*` style according to the coding style.

Reply:
done